receiver/telemetrygeneratorreceiver/windows_events_generator.go

```diff
@@ -15,24 +15,19 @@
te logon request originated. Workstation name is not always available and may be left blank in some cases."},{"stringValue":"The authentication information fields provide detailed information about this specific logon request."},{"stringValue":"- Transited services indicate which intermediate services have participated in this logon request."},{"stringValue":"- Package name indicates which sub-protocol was used among the NTLM protocols."},{"stringValue":"- Key length indicates the length of the generated session key. This will be 0 if no session key was requested."}]}}},{"key":"Subject","value":{"kvlistValue":{"values":[{"key":"Security ID","value":{"stringValue":"S-1-0-0"}},{"key":"Account Name","value":{"stringValue":"-"}},{"key":"Account Domain","value":{"stringValue":"-"}},{"key":"Logon ID","value":{"stringValue":"0x0"}}]}}}]}}},{"key":"event_id","value":{"kvlistValue":{"values":[{"key":"qualifiers","value":{"intValue":"0"}},{"key":"id","value":{"intValue":"4625"}}]}}},{"key":"event_data","value":{"kvlistValue":{"values":[{"key":"data","value":{"arrayValue":{"values":[{"kvlistValue":{"values":[{"key":"SubjectUserSid","value":{"stringValue":"S-1-0-0"}}]}},{"kvlistValue":{"values":[{"key":"SubjectUserName","value":{"stringValue":"-"}}]}},{"kvlistValue":{"values":[{"key":"SubjectDomainName","value":{"stringValue":"-"}}]}},{"kvlistValue":{"values":[{"key":"SubjectLogonId","value":{"stringValue":"0x0"}}]}},{"kvlistValue":{"values":[{"key":"TargetUserSid","value":{"stringValue":"S-1-0-0"}}]}},{"kvlistValue":{"values":[{"key":"TargetUserName","value":{"stringValue":"Administrator"}}]}},{"kvlistValue":{"values":[{"key":"TargetDomainName","value":{"stringValue":"-"}}]}},{"kvlistValue":{"values":[{"key":"Status","value":{"stringValue":"0xc000006d"}}]}},{"kvlistValue":{"values":[{"key":"FailureReason","value":{"stringValue":"%%2313"}}]}},{"kvlistValue":{"values":[{"key":"SubStatus","value":{"stringValue":"0xc000006a"}}]}},{"kvlistValue":{"values":[{"key":"LogonType","value":{"stringValue":"3"}}]}},{"kvlistValue":{"values":[{"key":"LogonProcessName","value":{"stringValue":"NtLmSsp "}}]}},{"kvlistValue":{"values":[{"key":"AuthenticationPackageName","value":{"stringValue":"NTLM"}}]}},{"kvlistValue":{"values":[{"key":"WorkstationName","value":{"stringValue":"workstation"}}]}},{"kvlistValue":{"values":[{"key":"TransmittedServices","value":{"stringValue":"-"}}]}},{"kvlistValue":{"values":[{"key":"LmPackageName","value":{"stringValue":"-"}}]}},{"kvlistValue":{"values":[{"key":"KeyLength","value":{"stringValue":"0"}}]}},{"kvlistValue":{"values":[{"key":"ProcessId","value":{"stringValue":"0x0"}}]}},{"kvlistValue":{"values":[{"key":"ProcessName","value":{"stringValue":"-"}}]}},{"kvlistValue":{"values":[{"key":"IpAddress","value":{"stringValue":"104.249.63.207"}}]}},{"kvlistValue":{"values":[{"key":"IpPort","value":{"stringValue":"0"}}]}}]}}}]}}}]}},"attributes":[{"key":"log_type","value":{"stringValue":"windows_event.security"}}],"traceId":"","spanId":""}]}]}]}`,
```

Comment:
Does this blob any any information that's unique to us? Was this gathered using the raw windows events receiver or the BPOP source? The BPOP source injects the resource detection processor.

Reply:
Ah interesting, I did use the BPOP source. I'll rerun it using just the raw win events receiver to be sure.